src/App.css

```diff
@@ -30,3 +30,34 @@
     transform: rotate(360deg);
   }
 }
+
+/* Calvin's CSS */
+
+a,
+a:hover,
+a:active,
+a:focus,
+a:visited {
+  color: #222;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+h3 {
+  margin: 0;
+}
+
+label {
+  display: block;
+}
+
+.whiskey-card {
+  margin: 0.5em;
+  padding: 0.5em;
+  border: solid 1px;
```

Comment:
Does `border` give a colored border by default if you don't specify?

Reply:
Defaults to black